Add a documentation page for most basic stuff

julienw opened the pull request:
This also fixes an issue with docsify affecting this new page, see https://github.com/docsifyjs/docsify/issues/1822 for more information.

[deploy preview](https://deploy-preview-4105--perf-html.netlify.app/docs/#/./guide-getting-started)

This also moves the content that used to live in "UI Tour: Sharing and Naming" to this new page.

canova commented on docs-user/guide-getting-started.md:
```diff
@@ -0,0 +1,114 @@
+
+![A screenshot highlighting the toolbar's upload profile button.](images/getting-started-upload.png)
+
+One of the most powerful and useful features of the Firefox Profiler is the ability to upload and share profiles. The first step is to hit the *Upload Local Profile* button. You'll be able to exclude some information before uploading. Then the profile will be uploaded to an online storage. This profile can then be shared in online chat, emails, and bug reports. Note that anybody that has the link ...
```
```suggestion
One of the most powerful and useful features of the Firefox Profiler is the ability to upload and share profiles. The first step is to hit the *Upload Local Profile* button. You'll be able to exclude some information before uploading. Then the profile will be uploaded to an online storage. This profile can then be shared in online chat, emails, and bug reports. Note that anybody that has the link will be able to access the uploaded data, as it's not protected otherwise.
```

canova commented on docs-user/guide-getting-started.md:
```diff
@@ -0,0 +1,114 @@
+
+![A screenshot showing the delete button in the profile info panel](images/getting-started-delete-profile.png)
+
+You can also go to the [Uploaded Recordings page](/uploaded-recordings/) to list
```
This link doesn't seem to work. It's trying to navigate to `/docs/uploaded-recordings/`

canova commented on docs-user/guide-getting-started.md:
```diff
@@ -0,0 +1,114 @@
+
+You can also go to the [Uploaded Recordings page](/uploaded-recordings/) to list
+all profiles you uploaded and delete them. Note that this list is stored locally
+in your Firefox only.
```
Do you think it's worth noting that you can only delete the profiles that you uploaded in that browser? (maybe also mentioning that they are stored in IndexedDB, but it might be too much detail for a getting started guide)

canova reviewed (approved):
Looks great to me! We usually take these stuff for granted and don't realize that getting started guide is very important. I think this is going to be very useful for our new users. Thanks for adding it!

julienw commented:
Thanks for the review, I fixed all your comments. I also added one last comment referencing the docsify pull request next to the script tag for the docsify file, so that if we update it in the future we don't forget about this fix.
